ccd-definition/AuthorisationCaseField/AuthorisationCaseField-HWF-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "caseTitleWithHWFClaimFeeNumber",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-admin"
```

Comment:
Should we add "caseworker-civil" as well here. I am not sure about roles but need to agree and make it consistent in HWF tickets